cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -714,6 +714,10 @@
       public static final String FILE_NAME = "security.store.file.name";
       /** Password to access the secure store. */
       public static final String FILE_PASSWORD = "security.store.file.password";
+      /** Backend provider for the secure store. e.g. file */
+      public static final String PROVIDER = "security.store.provider";
+      /** Separator between the namespace name and the key name */
+      public static final String NAME_SEPARATOR = ":";
```

Comment:
So we've started to only have parameter names from cdap-default.xml here. Is there any way users can change this? If not, better move this constant to FileSecureStore or some class that is only accessible to FileSecureStore and KMSSecureStore